analysis/include/BumpHunter.h

```diff
@@ -48,7 +48,7 @@ class BumpHunter {
         };
         
         /** Default Constructor */
-        BumpHunter(BkgModel model, int poly_order, int res_factor, bool asymptotic_limit = true);
+        BumpHunter(BkgModel model, int poly_order, int res_factor, double res_scale = 1.56, bool asymptotic_limit = true);
```

Comment:
Seems dangerous to me for this to default to anything except 1.

Reply:
1.56 was the correct default for Sebouh's resolution, but is not correct for Rafo's. I had updated it elsewhere but forgot to do it here. It should be fixed now.